shard.yml

```
@@ -18,6 +18,7 @@ dependencies:
     github: maiha/var.cr
   pretty:
     github: maiha/pretty.cr
+    version: 1.1.2
```

version指定良いですね。
固定だと「自動アップグレード」や「他shardとの連携」に難があるので、以上(`>=`)にしておく方が安全そうです。

```suggestion
    version: ">= 1.1.2"
```

---

ありがとうございます。